recirq/fermi_hubbard/publication.py

```diff
@@ -209,3 +213,50 @@ def rainbow23_layouts(sites_count: int = 8) -> Tuple[ZigZagLayout]:
                  for flip in (False, True)
                  for exchange in (False, True)
                  for reverse in (False, True))
+
+
+def fetch_publication_data(
```

Comment:
The other function is in `recirq.documentation_utils`. I don't know if that's the exact correct place to put it, but I think there's some value to having all the example-dataset functionality outside of the projects so you can find all of them more easily in one place. What do you think?

Reply:
I guess there are two options which make sense:

1. Put download functions in the same place (`documentation_utils.get_qaoa_data` and `documentation_utils.get_fermi_hubbard_data`).
2. Put download functions in respective modules (`qaoa.get_publication_data` and `recirq.get_publication_data`).

I'm not too partial to either so I'm good to go with your intuition here. Maybe (2) is slightly more natural to me since I would probably look there before `documentation_utils` if I were guessing. This could be improved by renaming `documentation_utils` or putting both functions in a new module though.